### DISCUSSION TITLE
Unable to download selected data

### DISCUSSION THREAD

**Discussion by S-Majoros** [overture-stack/docs#18] [Support] [answered]
Hello! On the Drug Discovery Data Portal I am able to download the full dataset, but when I select specific samples to download I am only given a blank file.

**Comment by justincorrigible** [comment]
This is likely caused by a Table configuration in need of adjustment, specifically the one that tells the table how to identify each individual row... usually accomplished by having a unique identifier in your data.

Is that piece of information available, and if so, in which column would it be? I'd be happy to guide on next steps once we have determined that

**Reply by S-Majoros** [reply]
There isn't one specific column that acts as the unique identifier. For the correlation table, the unique identifier is a combination of Hugo Symbol A, Hugo Symbol B, Cancer Type and Dataset Name. For the mutation frequency and mRNA tables, the identifier is a combination of Hugo Symbol, Cancer Type and Dataset Name. For the protein interactions table, the identifier is a combination of Hugo Symbol A, Hugo Symbol B and Dataset Name.

**Reply by MitchellShiell** [reply]
Worth mentioning I updated our [CSV-processor/uploader to generate and upload automated submitter metadata](https://github.com/overture-stack/conductor/blob/prelude/csv-processor/src/services/processor.ts#L129-L143) (Mimicking what Song does), with this we will have unique IDs generated and uploaded on submission (without user input required), once we update the drug discovery portal with this release we should be able to update our table configuration accordingly, will update here with more info!

**Comment by justincorrigible** [comment, marked as answer]
We've identified the issue was in a tricky part of the implementation within Arranger (both in the UI component and the Server side).
The issue has been rectified in v3-beta.36. @MitchellShiell will update the deployment accordingly.

Marking this as solved, and will reopen if needed.
